src/grt/src/fastroute/include/FastRoute.h

```diff
@@ -622,6 +621,7 @@ class FastRouteCore
 
   std::set<std::pair<int, int>> h_used_ggrid_;
   std::set<std::pair<int, int>> v_used_ggrid_;
+  std::vector<int> dirty_net_ids_;
```

Comment:
You can move this vector to the line after the `std::vector<FrNet*> nets_;` definition. Also, I think the name could be only `net_ids_`. FastRouteCore doesn't know about the "dirty nets" concept. It only makes sense in the GlobalRouter class.